packages/@react-aria/interactions/src/useInteractOutside.ts

```diff
@@ -62,6 +64,9 @@ export function useInteractOutside(props: InteractOutsideProps) {
       };
     } else {*/
     let onMouseUp = (e) => {
+      if (isDisabled) {
```

Comment:
this will not be updated if the prop `isDisabled` changes, we need to add it to the dep array for the useEffect like we do in useHover https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/src/useHover.ts#L181

Reply:
Fixed

b99db623